src/Shopsys/ShopBundle/Resources/config/services_test.yml

```diff
@@ -37,3 +37,11 @@ services:
       - { name: doctrine.event_subscriber, priority: -10 }
 
   League\Flysystem\FilesystemInterface: '@main_filesystem'
+
+  Shopsys\FrameworkBundle\Model\Category\CategoryFactoryInterface: '@Shopsys\ShopBundle\Model\Category\CategoryFactory'
```

Comment:
Please, consider to implement it the same way as Shopsys\FrameworkBundle\Model\Product\ProductFactoryInterface:
alias: Shopsys\ShopBundle\Model\Product\ProductFactory
since it is already merged into master

Reply:
Considered and applied